content/docs/user-guide/integrations/huggingface.md

```diff
@@ -0,0 +1,119 @@
+# Hugging Face
+
+DVC includes native support for importing and downloading data hosted on the
+[Hugging Face Hub](https://huggingface.co/docs/hub/index). Data from a <abbr>DVC
+project</abbr> can also be loaded with the Hugging Face
+[Datasets](https://huggingface.co/docs/datasets/index) library via the DVC
+[Python API](/doc/api-reference).
+
+## Hugging Face Hub
+
+DVC supports importing or downloading data from any Git repository, even
+repositories which use [Git-LFS](https://git-lfs.com/) like repositories hosted
+on the [Hugging Face Hub](https://huggingface.co/docs/hub/index). Both HTTPS
+(`https://huggingface.co/<MODEL>` and SSH (`git@hf.co:<MODEL>`) repository URLs
+are supported.
+
+<admon type="tip">
+
+DVC already includes native read-only Git-LFS support, so using DVC to import
+and download data from Hugging Face Hub does not require installing Git LFS or
+the Hugging Face CLI.
+
+</admon>
+
+<admon type="info">
+
+When using Git over SSH with Hugging Face Hub (i.e. `git@hf.co:<MODEL>` URLs)
+you must have a Hugging Face account with a registered SSH key, regardless of
+whether the repository is public or private. Using Git over HTTPS to access
+public repositories does not require a Hugging Face account. Please refer to the
+Hugging Face [documentation](https://huggingface.co/docs/hub/security-git-ssh)
+for more information.
+
+This limitation applies to both `dvc get` and `dvc import`.
+
+</admon>
+
+### Setup
+
+```cli
+$ pip install dvc
+```
+
+### Download data
```

Comment:
It would be good to explain the use cases for each of `dvc get` and `dvc import` with hf hub. For example, `dvc get` to download data without having to clone the repo or write a python script. `dvc import` is useful for using the dataset in a downstream project.